From 34bb581e7713589d38c797c214f4c6bf2b14b702 Mon Sep 17 00:00:00 2001
From: Florian Weimer <fweimer@redhat.com>
Date: Fri, 16 Aug 2024 16:05:19 +0200
Subject: [PATCH] support: Include <string.h> for strcmp in
 support_format_addrinfo.c
Content-type: text/plain; charset=UTF-8

This is currently implied by the internal headers, but it makes
sense not to rely on this.

Reviewed-by: Adhemerval Zanella <adhemerval.zanella@linaro.org>
---
 support/support_format_addrinfo.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/support/support_format_addrinfo.c b/support/support_format_addrinfo.c
index cbc72910a9..77f4db345c 100644
--- a/support/support_format_addrinfo.c
+++ b/support/support_format_addrinfo.c
@@ -22,6 +22,7 @@
 #include <errno.h>
 #include <stdio.h>
 #include <stdlib.h>
+#include <string.h>
 #include <support/support.h>
 #include <support/xmemstream.h>
 
-- 
2.43.5

